cl/search/api_utils.py

```diff
@@ -262,3 +500,36 @@ def __getattr__(self, key):
 
     def to_dict(self):
         return self._data
+
+
+class ESResultObject(ResultObject):
+
+    def __getattr__(self, key):
+        return getattr(self._data, key, None)
+
+
+def limit_api_results_to_page(
+    results: Response | AttrList, cursor: ESCursor
+) -> Response | AttrList:
+    """In ES Cursor pagination, an additional document is returned in each
+    query response to determine whether to display the next page or previous
+    pages. Here we limit the API results to the number defined in
+    settings for a single page, according to the navigation action being
+    performed.
+
+    :param results: The results returned by ES.
+    :param cursor: A ESCursor instance containing the "search_after" parameter
+     and a boolean "reverse" indicating if going backwards.
+    :return: A slice of the results list, limited to the number of items as
+    specified by the SEARCH_API_PAGE_SIZE.
+    """
+
+    reverse = False
+    if cursor is not None:
+        search_after, reverse, _ = cursor
```

Comment:
We're not using the `search_after` variable here. I think we can simplify this using a ternary if statement 

```suggestion
    reverse = cursor.reverse if cursor else False
```

Reply:
Yeah, this is better, changed it!